nginx.tmpl

```diff
@@ -1,3 +1,5 @@
+{{ $CurrentContainer := where $ "ID" .Docker.CurrentContainerID | first }}
```

Comment:
I just realized this will break in the case that docker-gen is in a
different container from nginx and the two are not on the same networks.
Definitely an edge case, but it should at least be covered in the docs if
it isn't handled correctly.